libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFSecurityLockout.m

```diff
@@ -75,39 +75,41 @@ @implementation SFSecurityLockout
 
 + (void)initialize
 {
-    [SFSecurityLockout upgradeSettings];  // Ensures a lockout time value in the keychain.
-    
-    // If this is the first time the passcode functionality has been run in the lifetime of the app install,
-    // reset passcode data, since keychain data can persist between app installs.
-    if (![SFCrypto baseAppIdentifierIsConfigured] || [SFCrypto baseAppIdentifierConfiguredThisLaunch]) {
-        [SFSecurityLockout setSecurityLockoutTime:0];
-        [SFSecurityLockout setPasscodeLength:kDefaultPasscodeLength];
-    } else {
-        securityLockoutTime = [[SFSecurityLockout readLockoutTimeFromKeychain] unsignedIntegerValue];
-    }
-    
-    sDelegates = [NSMutableOrderedSet orderedSet];
-    
-    [SFSecurityLockout setPasscodeViewControllerCreationBlock:^UIViewController *(SFPasscodeControllerMode mode, SFPasscodeConfigurationData configData) {
-        SFPasscodeViewController *pvc = nil;
-        if (mode == SFPasscodeControllerModeCreate) {
-            pvc = [[SFPasscodeViewController alloc] initForPasscodeCreation:configData];
-        } else if (mode == SFPasscodeControllerModeChange) {
-            pvc = [[SFPasscodeViewController alloc] initForPasscodeChange:configData];
+    if (self == [SFSecurityLockout class]) {
```

Comment:
Added this for sanity's sake, though heaven help us if anyone subclasses this. 😳